src/store/reducers/dform/initial/styles.json

```diff
@@ -1841,11 +1865,80 @@
+          "label": "Serological Tests",
+          "show": "test-order:serological",
+          "options": [
+            {"id": "pathology.serological.HIV",
+            "label": "HIV I AND II antibody"},
+            {"id": "pathology.serological.HBsAg",
+            "label": "HBsAg"},
+            {"id": "pathology.serological.AntiHCV", "label": "Anti HCV"},
+            {"id": "pathology.serological.VDRL", "label": "VDRL"},
+            {"id": "pathology.serological.TPHA", "label": "TPHA"},
+            {"id": "pathology.serological.ABOgrouping", "label": "ABO Grouping"},
+            {"id": "pathology.serological.RhTyping", "label": "Rh Typing"},
+            {"id": "pathology.serological.RAFactor", "label": "RA Factor"},
+            {"id": "pathology.serological.CRP", "label": "CRP"},
+            {"id": "pathology.serological.ASO", "label": "ASO"}
+          ]
+        },
+        {
+          "field": "semenanalysis",
+          "class": "checkgroup",
+          "label": "Semen Analysis",
+          "show": "test-order:semenanalysis",
+          "options": [
+            {"id": "pathology.SemenAnalysis.CollectionTime", "label": "Collection Time"},
+            {"id": "pathology.SemenAnalysis.ExaminationTime","label": "Examination Time"},
+            {"id": "pathology.SemenAnalysis.Volume", "label": "Volume"},
+            {"id": "pathology.SemenAnalysis.PH", "label": "PH"},
+            {"id": "pathology.SemenAnalysis.SpermatozoaMotility", "label": "Spermatozoa Motility"},
+            {"id": "pathology.SemenAnalysis.SpermatozoaCount", "label": "Spermatozoa Count"},
+            {"id": "pathology.SemenAnalysis.SpermMorpholohy", "label": "Sperm Morphology"},
+            {"id": "pathology.SemenAnalysis.Visocity", "label": "Viscosity"},
+            {"id": "pathology.SemenAnalysis.OtherCell", "label": "Other Cell"}
+          ]
+        },
+        {
+          "field": "mantoux",
+          "class": "checkgroup",
+          "label": "Mantoux Test",
+          "show": "test-order:mantoux",
+          "options": [
+            {"id": "pathology.mantouxTest.Intro", "label": "Introduction"},
+            {"id": "pathology.mantouxTest.Erythroma", "label": "Erythroma"},
+            {"id": "pathology.mantouxTest.Result", "label": "Result"}
+          ]
+        },
+        {
+          "field": "gynecology",
+          "class": "checkgroup",
+          "label": "Gynecology",
+          "show": "test-order:gynecology",
+          "options": [
+            {"id": "pathology.Pap_smear_report", "label": "Pap smear report"},
+            {"id": "pathology.gynecology_description", "label": "Gynecology Description"},
+            {"id": "pathology.gynecology_comment",
+            "label": "Gynecology Comment"
+            }
+          ]
+        },
         {
           "field": "tests",
           "label": "Free description",
-          "class": "textarea",
-          "show": "plan:test"
+          "class": "checkgroup",
```

Comment:
It seems this shouldn't have been changed to 'checkgroup'.
`class: "checkgroup"` with empty `options` shows nothing:
<img width="675" alt="2018-09-24 13 34 53" src="https://user-images.githubusercontent.com/3135397/45937793-aa073e80-bffe-11e8-8123-62736fb31665.png">